plugin/src/main/kotlin/com/ritense/plugin/service/PluginService.kt

```diff
@@ -536,6 +536,14 @@ class PluginService(
         return pluginConfiguration?.let { createInstance(it) as T }
     }
 
+    fun <T> createInstances(clazz: Class<T>): List<T> {
```

Comment:
Why was this added if it's not used? And if it's needed: can we keep the filter argument in?

Reply:
Changed